src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -425,19 +425,6 @@ public void TestUInt32TypeCodes()
             Assert.Equal(ObjectWriter.TypeCode.UInt32_10, ObjectWriter.TypeCode.UInt32_0 + 10);
         }
 
-        private void TestRoundTripType(Type type)
-        {
-            TestRoundTrip(type, (w, v) => w.WriteType(v), r => r.ReadType());
-        }
-
-        [Fact]
-        public void TestTypes()
-        {
-            TestRoundTripType(typeof(int));
-            TestRoundTripType(typeof(string));
-            TestRoundTripType(typeof(ObjectSerializationTests));
-        }
```

Comment:
Why was this test removed?

Reply:
These tests were the only usage I found of the object serialization functionality.

Reply:
Presumably though this is a feature that is in use outside of trimming though yes?